src/pages/news/2020-05-25-bitcoin-tren-eth-chi-la-toan-hoc.vn.md

```diff
@@ -0,0 +1,86 @@
+
+Mạng Keep là một mạng off-chain gồm các nút đọc và ghi vào mạng Ethereum và Bitcoin. Trong hệ thống tBTC, những người ký tên phải đặt tài sản thế chấp bằng 1,5 lần số Bitcoin được gửi trong ETH, để đảm bảo chống lại hoạt động làm hại.
+
+![bai4.jpeg](https://cdn.steemitimages.com/DQmNLrDdH8koZcCJvqScsnb1bGpk2813MRAh1QMTbzaXJAE/bai4.jpeg)
+
+### Đổi TBTC lấy Bitcoin
+
+Mua lại đơn giản như tạo tiền; 1 TBTC luôn được đổi thành 1 BTC. Khi bạn sẵn sàng chuyển trở lại BTC, bạn có thể sử dụng Token tiền gửi tBTC (TDT) của mình để đổi UTXO chính xác của khoản tiền gửi ban đầu nếu trong thời hạn gửi 6 tháng (người ký được trả phí tại cuối kỳ). Cách thể đổi TBTC lấy Bitcoin:
+
+•	Gửi yêu cầu đổi TBTC, cùng với token Tiền gửi tBTC.
+
+•	Nếu bạn sở hữu TDT cho khoản tiền gửi, bạn cũng cần gửi một khoản phí người ký (tức là nếu bạn đã tạo 1 TBTC, bạn sẽ gửi yêu cầu đổi với 1.005 TBTC). Điều này sẽ cung cấp cho bạn UTXO chính xác được liên kết với TDT của bạn.
+
+•	Nếu bạn không sở hữu TDT nhưng bạn muốn đổi bitcoin cho TBTC của mình, bạn chỉ cần trả phí người ký cộng với TBTC để đổi BTC từ hệ thống.
+
+•	Nếu bạn không sở hữu TDT, nhưng có một khoản tiền gửi đã tồn tại> 6 tháng trong hệ thống, bạn sẽ chỉ mất 1 TBTC để đổi lấy 1 BTC. Trong trường hợp này, phí người ký được trả bởi chủ sở hữu tiền gửi tBTC hiện tại, nhưng bạn (người đổi) nhận được BTC. 
+
+•	Hệ thống keep sẽ yêu cầu bạn xác nhận yêu cầu BTC của bạn trên chuỗi khối Ethereum. 
+
+•	Hệ thống phát hành BTC và xác nhận việc phát hành trên chuỗi ETH.
+
+•	Sau khi hoàn thành BTC, hệ thống Keep sẽ gửi bằng chứng giao dịch BTC đến chuỗi ETH, kích hoạt thanh toán cho người ký. 
+
+### Cập nhật phát triển tBTC
+
+
+![bai5.png](https://cdn.steemitimages.com/DQmZXYuEzDdYifHdsrQkvBoq8dMNrjsCMwfagVtV37crvmc/bai5.png)
+
+#### Quy trình đổi:
+
+Ứng dụng đổi cho phép một khoản tiền gửi tBTC được đổi lại cho các khoản BTC ban đầu bằng Token tiền gửi tBTC hợp lệ. Một TDT được gắn với một khoản tiền gửi tBTC cụ thể Bitcoin Bitcoin UTXO. Giữ token cho phép đổi số tiền gửi tBTC sang Bitcoin ban đầu.
+
+Trong các thiết kế ban đầu, tiền gửi tBTC ngay lập tức được chuyển đổi thành token ERC-20 TBTC khi đủ điều kiện (bằng chứng tiền gửi). Trong thiết kế cuối cùng của tBTC v1, theo mặc định, tiền gửi đủ điều kiện chỉ cung cấp cho bạn một TDT. Có một hợp đồng riêng, được gọi là Máy bán hàng tự động TBTC, được sử dụng để tạo TBTC để đổi lấy TDT, trừ đi một khoản ký quỹ bằng với phí ký tên cho khoản tiền gửi. Chủ sở hữu TDT có quyền độc quyền mua lại UTXO cơ bản cho đến khi khoản tiền gửi vượt thời hạn (6 tháng), trong khi tạo TBTC từ tiền gửi chuyển quyền sở hữu token đó sang máy bán hàng tự động, từ đó mọi người có thể lấy và đổi tiền gửi . 
+
+DeFi mở rộng quy mô với Ethereum, nhưng hãy tưởng tượng điều gì sẽ xảy ra khi DeFi có quyền truy cập vào thanh khoản Bitcoin Bitcoin. Sắp ra mắt: tBTC đã chạy trên mạng kiểm tra nội bộ của chúng tôi kể từ tháng 10 và sẽ có trên mạng kiểm tra công khai vào đầu Q1.
```

Comment:
Seems like the "Bitcoin Bitcoin" here is a typo?

Reply:
Fixed duplicate issue.